### ISSUE TITLE
Should this repo go under electron-userland?

### ISSUE THREAD

**Issue by amaurymartiny** [maxogden/menubar#211]
From https://github.com/electron-userland/welcome

> This is an organization for modules that meet the following criteria:
    - Want to use http://openopensource.org/ governance
    - Would benefit from having neutral ownership (e.g. anyone can become a maintainer)
    - Provide a generally useful function to the electron community
    - Would otherwise not be maintained effectively

I believe this repo ticks the 4 boxes, so in my opinion it should go under electron-userland. But I would like to hear your opinion too.

**Comment by cyberious** [comment]
This is even more relevant now that it looks to have been abandoned.

**Comment by amaurym** [comment]
What are you saying it's abandoned? Last commit was last month.

**Comment by cyberious** [comment]
By Dependabot! Do you consider that active development?

**Comment by amaurym** [comment]
Well, I merged that PR manually, after assessing that it was a safe and useful dep upgrade.

There has been [regular commits](https://github.com/maxogden/menubar/graphs/contributors) to this repo since I took over in 2019. The 2017-2019 period could be called abandoned.

**Comment by setchy** [comment]
@amaury1093 - as per your original post way back in 2019, do you think it is worth re-exploring the transition of this repository to https://github.com/electron-userland? 

I'm happy to assist where possible

**Comment by amaury1093** [comment]
Yes, please take the lead to transferring this repo to electron userland. That's my preferred way moving forward. 

Do note that it's ultimately not my decision, I only have push/write rights to this repo, and no admin rights. I also have a hard time getting in touch with the author.
